extensions/amp-nexxtv-player/0.1/validator-amp-nexxtv-player.protoascii

```diff
@@ -49,21 +49,21 @@ tags: {  # amp-nexxtv-player
       error_message: "contents"
     }
   }
-  spec_url: "https://www.ampproject.org/docs/reference/extended/amp-nexxtv-player.html"
+  spec_url: "/extensions/amp-nexxtv-player/amp-nexxtv-player.md"
```

Comment:
Thanks for changing this, but we'll want it to be an absolute URL.
`https://github.com/ampproject/amphtml/blob/master/extensions/amp-nexxtv-player/amp-nexxtv-player.md`